backend/program/scrapers/torrentio.py

```diff
@@ -1,26 +1,21 @@
 """ Torrentio scraper module """
 from typing import Optional
-from pydantic import BaseModel
+
 from requests import ConnectTimeout, ReadTimeout
 from requests.exceptions import RequestException
+
 from utils.logger import logger
 from utils.request import RateLimitExceeded, get, RateLimiter, ping
-from utils.settings import settings_manager
+from program.settings.manager import settings_manager
 from utils.parser import parser
 
 
-class TorrentioConfig(BaseModel):
-    enabled: bool
-    url: Optional[str]
-    filter: Optional[str]
-
-
 class Torrentio:
     """Scraper for `Torrentio`"""
 
     def __init__(self, _):
         self.key = "torrentio"

```

Comment:
probably would be better to just rely on class naming convention instead of these key strings

Reply:
I believe they were being used in fetching settings? So you could update one name instead of going through the module and replacing all the cases.. 

Also, it may be used elsewhere for logging purposes if I'm not mistaken